### PR TITLE
Moved Change User Password and added password length check

### PR DESCRIPTION
**Pull request by jinapurapu**
**What does this do?**
Feature to change password for selected user moved to an Iconbutton on UserDetails.tsx, along with additional delete user function.
Password length checking with error message added to ChangeUserPassword, AccountChangePassword and AddUser modals   

**How does it look?**
 
Screenshot from 2021-06-17 15-29-18
Screenshot from 2021-06-17 15-29-37
Screenshot from 2021-06-17 15-30-03

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]
tested. LGTM

**Review by bexsoft — COMMENTED** [review]
LGTM, waiting for Daniel's merge to approve

File: portal-ui/build/asset-manifest.json [hunk 1]
```diff
@@ -1,25 +1,139 @@
...
+    "main.js": "/static/js/main.f0be6105.chunk.js",
+    "main.js.map": "/static/js/main.f0be6105.chunk.js.map",
+    "runtime-main.js": "/static/js/runtime-main.43a31377.js",
+    "runtime-main.js.map": "/static/js/runtime-main.43a31377.js.map",
+    "static/css/2.c5a51b70.chunk.css": "/static/css/2.c5a51b70.chunk.css",
+    "static/js/2.8df217ff.chunk.js": "/static/js/2.8df217ff.chunk.js",
+    "static/js/2.8df217ff.chunk.js.map": "/static/js/2.8df217ff.chunk.js.map",
+>>>>>>> fedb8b1... Checked out build files
+>>>>>>> f1114e5... Moved Change User Password Modal  to UserDetails.tsx
+>>>>>>> dc398d193f6788d7bef7a01eeb8bb797542fc503
```
Comment (Alevsk) [inline portal-ui/build/asset-manifest.json:56]: This has some unresolved merge conflicts, you can fix this by running: `make assets`

**Review by Alevsk — CHANGES_REQUESTED** [review]

**Review by dvaldivia — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by Alevsk — APPROVED** [review]

**Pull request merged**